modules/codegen/src/main/scala/com/twilio/guardrail/generators/ScalaGenerator.scala

```diff
@@ -180,6 +180,7 @@ object ScalaGenerator {
                 implicit val showLocalDate = build[java.time.LocalDate](_.format(java.time.format.DateTimeFormatter.ISO_DATE))
                 implicit val showOffsetDateTime = build[java.time.OffsetDateTime](_.format(java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME))
                 implicit val showJavaURL = build[java.net.URI](_.toString)
+                implicit def showIterable[K](implicit s: Show[K]) = build[Iterable[K]](_.map(s.show).mkString(","))
```

Comment:
This is very undesirable, as it establishes a new, stringy protocol.

If I try to `Show[String]` of `List("foo", "bar,baz", "blix")`, then the consumer attempts to derive the members again, I'd be forced to `values.split(",")`, which would give me `Array("foo", "bar", "baz", "blix")`, which does not round-trip.

Why is this necessary? If it's for query string or form parameters, we should expand into successive single values, like `Foo=1&Foo=2&Foo=3`

Reply:
This was done for header values. In case we have a multi-value header we apply this implicit. This is used only for akka-http as far as I understood